cppcache/integration-test/testThinClientRemoteQueryTimeout.cpp

```diff
@@ -290,7 +291,7 @@ DUNIT_TASK_DEFINITION(CLIENT1, StepSix)
     try {
       LOG("EXECUTE 4 START");
 
-      results = qry->execute(std::chrono::seconds(850));
+      results = qry->execute(std::chrono::seconds(850000));
```

Comment:
850000 seconds is approximately 9.8 days.  Seems a little extreme as a timeout value, you probably wanted milliseconds